README.md

```diff
@@ -130,19 +130,109 @@ public class Example {
 
 It may be useful to check the test implementation in the folder **test** `(./src/test)`
 
-## 🐳 Running local tests with docker
+## 🐳 Running local tests automatically -  with docker
 
 1. Copy `.env.example` to `.env` and fill in the values.
 2. Run `docker build -t java .` to build the image.
 3. Run `docker run --rm java` to run the tests and delete container immediately after.
 
+## Running local tests manually - with commands
+
+1. Make sure you have OpenJDK (desired version) installed (via Homebrew)
+2. From the main java folder run `mvn test`
+
 ## 🛠️ Contributing
 
 Read more about how to Contribute to Voucherify Java SDK by visiting main repo [GENERATING-SDKS.md](https://github.com/voucherifyio/voucherify-openapi/blob/master/GENERATING-SDKS.md)
 
 Remember that this SDK is auto generated (except of the tests) so changes made here will be overwritten by generator.
 
+## Link tags
+[OpenAPI generated from tag](https://github.com/voucherifyio/voucherify-openapi/releases/tag/sdk-java-17.0.0)
+
 ## 📅 Changelog
+- **2025-07-06** - `17.0.0`
+ADDED:
+  - ClientQualificationsCheckEligibilityResponseBodyOrder model for order object in ClientQualificationsCheckEligibilityResponseBody model
+  - ClientRedemptionsRedeemResponseBodyOrder model for order object in ClientRedemptionsRedeemResponseBody model
+  - ClientValidationsValidateResponseBodyRedeemablesItemOrder model for order object in ClientValidationsValidateResponseBody model
+  - ClientValidationsValidateResponseBodyOrder model for order object in ClientValidationsValidateResponseBody model
+  - LoyaltiesMembersRedemptionRedeemResponseBodyOrder model for order object in LoyaltiesMembersRedemptionRedeemResponseBody model
+  - ParameterCampaignStatusListCampaigns model
+  - ParameterCampaignsAccessSetttings model
+  - ParameterFiltersListMemberTransactions model
+  - ParameterIsReferralCodeListCampaigns model
+  - ParameterPage99 model
+  - QualificationsCheckEligibilityResponseBodyOrder model for order object in QualificationsCheckEligibilityResponseBody model
+  - RedemptionsGetResponseBodyOrderItemsItem model for items object in RedemptionsGetResponseBody model
+  - `target` enum in ApplicableTo model
+  - `operation_status` enum in AsyncActionBase model
+  - `$contains`, `$not_contains` in FilterConditionsString model
+  - `properties` in MemberActivity model
+  - EventSource model 
+  - `target` enum in InapplicableTo model
+  - MemberActivityData model
+  - ListPublicationsItemValidSingleVoucher model
+  - SimpleCampaign model
+  - SimpleCampaignVoucher model
+  - `units_limit_exceeded` in ApplicableTo model
+  - `limit` in Bundle and ValidationRuleBundleRules models
+
+UPDATED:
+  - ParameterFiltersListCampaigns model:
```

Comment:
imo this shall be a parto of "added" - added new properties to existing models, imo no need to describe "(ParameterFiltersListCampaignsCampaigns model) as NEW"